locations/spiders/moneygram.py

```
@@ -1,41 +1,249 @@
+            location.update(location.pop("locationDetails"))
+            item = DictParser.parse(location)
+            item["branch"] = item.pop("name")
+            item["ref"] = " - ".join([item["ref"], kwargs["country"]])
+            try:
+                oh = OpeningHours()
+                for key, value in location.get("locationOperatingHours").items():
+                    day = key.replace("Hours", "")
+                    open_time, close_time = value.split(" - ")
+                    oh.add_range(day=day, open_time=open_time, close_time=close_time, time_format="%I:%M %p")
+                item["opening_hours"] = oh
+            except:
```

Ruff (0.16.1)

[error] Do not use bare `except`

(E722)